web/src/topic_list_data.ts

```diff
@@ -187,7 +187,13 @@ export function get_list_info(
     }
 
     if (zoomed) {
-        topic_names = util.filter_by_word_prefix_match(topic_names, search_term, (item) => item);
+        const word_separator_regex = /[\s/_-]/; // Use -, _, / as word separators in addition to spaces.
```

Comment:
This seems to be a slightly different character list than the existing copy in `strema_list_sort`; is that intentional?

```
    // Use -, _, : and / as word separators apart from the default space character                      
    const word_separator_regex = /[\s/:_-]/;                                                            
    stream_ids = util.filter_by_word_prefix_match(                                                      
        stream_ids,                                                                                     
        search_term,                                                                                    
        stream_id_to_name,                                                                              
        word_separator_regex,                                                                           
    );                                                                                                  
```

Reply:
Good observation. The colon (:) wasn't initially mentioned in the requirements, but after further consideration, I realized it could improve the user experience by serving as an additional word separator. I've updated my PR accordingly to include it.

Reply:
Why is this not (a) discussed in the linked CZO thread for the issue, and (b) called out in your PR description? You even marked "Explains differences from previous plans (e.g., issue description)." in the self-review checklisst.

Reply:
Clear communication is absolutely critical for continuing to have your work reviewed; silently making your code implement something different from the spec is not an effective way to propose changes to the spec.

Reply:
You're absolutely right, and I apologize for the oversight.

Reply:
Oh, Tim tells me that channel names actually use `:` as well; I think this got changed as a follow-up to the channels issue I copied.

Given that, I think it's fine to add a `:` here as well (but noting it on the PR description + CZO thread is important).